master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.service.deploy.master
+
+import org.scalatest.funsuite.AnyFunSuite
+
+import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.CelebornConf.{MASTER_HOST, MASTER_PORT}
+import org.apache.celeborn.common.internal.Logging
+import org.apache.celeborn.common.util.Utils
+
+class MasterArgumentsSuite extends AnyFunSuite with Logging {
+
+  test("Test build masterArguments") {
```

Comment:
Lets also add `CELEBORN-98: xxx`

Reply:
OK, i'd like add more info to test case name